pipelines/coaddQAPlots.yaml

```diff
@@ -0,0 +1,234 @@
+description: Make coadd plots for QA
+parameters:
+  ApFluxName: "iApFlux"
+  band: i
+tasks:
+  plot_CircAp12_sub_PS_meas_sky_stars:
+    class: lsst.analysis.drp.skyPlot.SkyPlotTask
```

Comment:
I'm a little surprised *ALL* of the plot tasks use the same class. I guess if you name it differently, you're allowed to use the same class multiple times in the same pipeline? Neat!

Reply:
Currently all the plots are the same type but more of different types will be added with time. This is mostly just getting a skeleton in place while we develop more plots to add.